lib/api/core/statistics.js

```diff
@@ -145,6 +145,55 @@ module.exports = function (kuzzle) {
   };
 
   /**
+   * Gets the lasts saved statistics frame from a date
+   *
+   * @param requestObject
+   * @returns {Promise} promise resolved as a ResponseObject
+   */
+  this.getStats = function(requestObject) {
+    var
+      deferred = q.defer(),
+      response = {
+        statistics: {}
+      },
+      frames,
+      currentDate,
+      since;
+
+    currentDate = new Date().getTime();
+    since = requestObject.data.body.since;
+    if (since > currentDate) {
+      deferred.resolve(new ResponseObject(requestObject, response));
+      return deferred.promise;
+    }
+    if (!this.lastFrame) {
+      if (!since || currentDate >= new Date(since).getTime()) {
```

Comment:
the second condition is always ``false`` as it has already been evaluated line 165